### DISCUSSION TITLE
Hvilke tiltak må innføres for å ivareta konfidensialiteten?

### DISCUSSION THREAD

**Discussion by troelde** [NorskHelsenett/Tillitsrammeverk#48] [General]
Ref. 6.2.1 Lekkasje av sensitive personopplysninger

"_For å sikre oss mot potensiell lekkasje av PII bør det vurderes å innføre tiltak for å ivareta konfidensialiteten._"

JWT og SAML-tokens transporteres i en HTTPS request i hhv. HTTP Authorization Header og HTTP Body som Header i SOAP-melding. Sistnevnte er da like godt/dårlig beskyttet som selve payload i SOAP-meldingen. For førstnevnte, så er det jo en generell utfording av HTTPS termineres i transportlaget. Det gjelder jo både HTTP Headers og Body. Må man man ha ende-til-ende-kryptering, så er vel VPN eneste mulighet dersom man ikke implementerer dette i applikasjonslaget (JWE) og kryptering av payload i HTTP Body.

**Comment by richardhus** [comment]
Støtter denne vurderingen, @troelde, det eneste punktet der vi ikke samtidig har payload i klartekst som følge av terminert HTTPS er vel selve STS-en, så da er det nok den man må vurdere spesielt heller enn kjeden der det er vel så problematisk at payload ikke er kryptert på termineringspunktene samtidig som det ligger tilsvarende opplysninger tilgjengelige i et JWT- eller SAML-sikkrehetsbevis.
